src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,259 @@
+
+All the functions below take into account the following options, so the returned paths will change based on their values:
+
+- [`base`](/en/reference/configuration-reference/#base)
+- [`trailingSlash`](/en/reference/configuration-reference/#trailingslash)
+- [`build.format`](/en/reference/configuration-reference/#buildformat)
+- [`site`](/en/reference/configuration-reference/#site)
+
+If `i18n.routingStrategy` is equals to `'prefix-other-locales'`, when the functions will process the `i18n.defaultLocale` when passed as input, the returned URLs **won't** contain the `i18n.defaultLocale`.  
+
+### `getRelativeLocaleUrl()`
+
+
+`getRelativeLocaleUrl(locale: string, path: string, options?: GetLocaleOptions): string`
+
+Use this function to retrieve a relative path for a locale. If the locale doesn't exist, Astro throws an error. 
+
+```astro title="src/pages/en/index.astro"
+---
+import {getRelativeLocaleUrl} from "astro:i18n";
+
+getRelativeLocaleUrl("fr", ""); // returns /fr
+getRelativeLocaleUrl("fr", "getting-started"); // returns /fr/getting-started
+getRelativeLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog"
+}); // returns /blog/fr-ca/getting-started
+
+getRelativeLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog",
+  normalizeLocale: false
+}); // returns /blog/fr_CA/getting-started
+
+---
+```
+
+### `getAbsoluteLocaleUrl()` 
+
+`getAbsoluteLocaleUrl(locale: string, path: string, options?: GetLocaleOptions): string`
+
+
+Use this function to retrieve an absolute path for a locale _if [`site`] has a value_. If [`site`] isn't, the function returns a relative URL. If the locale doesn't exist, Astro throws an error.
+
+
+```astro title="src/pages/en/index.astro"
+---
+import {getAbsoluteLocaleUrl} from "astro:i18n";
+
+// If `site` is set to be `https://example.com`
+
+getAbsoluteLocaleUrl("fr", ""); // returns https://example.com/fr
+getAbsoluteLocaleUrl("fr", "getting-started"); // returns https://example.com/fr/getting-started
+getAbsoluteLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog"
+}); // returns https://example.com/blog/fr-ca/getting-started
+
+getAbsoluteLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog",
+  normalizeLocale: false
+}); // returns https://example.com/blog/fr_CA/getting-started
+
+---
+```
+
+### `getRelativeLocaleUrlList()`
+
+It works like [`getRelativeLocaleUrl`](#getrelativelocaleurl), so it returns a list of relative paths for all the locales.
+
+
+`getRelativeLocaleUrlList(locale: string, options?: GetLocaleOptions): string[]`
+
+### `getAbsoluteLocaleUrlList()`
+
+`getAbsoluteLocaleUrlList(locale: string, options?: GetLocaleOptions): string[]`
+
+
+It works like [`getAbsoluteLocaleUrl`](#getabsolutelocaleurl), so it returns a list of relative paths for all the locales.
+
+
+## Routing strategy 
+
+The option `i18n.routingStrategy` allows to change the behaviour of the Astro routes
```

Comment:
```suggestion
The option `i18n.routingStrategy` allows changing the behaviour of the Astro routes.
```

Maybe `router` at the end, and not routes? Might be worth doing a rephrase here...